addons/medical/functions/fnc_addDamageToUnit.sqf

```diff
@@ -49,7 +44,7 @@ private _startDmg = +(_unit getVariable [QEGVAR(medical,bodyPartDamage), [0,0,0,
 private _startPain = GET_PAIN(_unit);
 #endif
 
-[QEGVAR(medical,woundReceived), [_unit, _bodyPart, _damageToAdd, _instigator, _typeOfDamage, _damageSelectionArray]] call CBA_fnc_localEvent;
+[QEGVAR(medical,woundReceived), [_unit, [[_bodyPart, _damageToAdd]], _instigator, _typeOfDamage]] call CBA_fnc_localEvent;
```

Comment:
```suggestion
[QEGVAR(medical,woundReceived), [_unit, [[_damageToAdd, _bodyPart, _damageToAdd]], _instigator, _typeOfDamage]] call CBA_fnc_localEvent;
```
Simplest fix.

Reply:
Good catch, thanks. I changed the format of woundReceived to also take the damage-before-armor value and apparently forgot to update it everywhere.